woocommerce/woocommerce-gateway-stripe
# feat: add statement-descriptor-error-message

# Changes proposed in this Pull Request:

When saving the settings, in WCPay we show a message when the statement descriptor fails validation.
Let's do the same here 🤷 

![2021-10-08 15 20 35](https://user-images.githubusercontent.com/273592/136621677-fd330939-f3ff-489c-8112-620428f4d90c.gif)

![2021-10-08 15 23 05](https://user-images.githubusercontent.com/273592/136621668-e976b344-1e4a-4f07-b01a-5020a04c25c5.gif)


# Testing instructions
- Ensure you have the `_wcstripe_feature_upe_settings` flag enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe&panel=settings
- Enter a less-than-5-characters statement descriptor (or add some special characters to it, like `'` or `"`)
- Try and save the settings
- Error appears
- Edit it
- Error disappears
- It's the same behavior as WCPay
